### PR TITLE
Prep cloud.common 3.0.0 release

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Prep cloud.common 3.0.0 release

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

several

### CODE REVIEW

**Comment by alinabuzachis** [comment]
recheck

**Review by abikouo — APPROVED** [review]

**Comment by alinabuzachis** [comment]
recheck

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -156,3 +156,15 @@ releases:
     - release_summary.yml
     - update_ansible_version.yml
     release_date: '2023-08-14'
+  3.0.0:
+    changes:
+      breaking_changes:
+      - Bump minimum Python suupported version to 3.9.
```
Comment (hakbailey) [inline changelogs/changelog.yaml:162]:
```suggestion
      - Bump minimum Python supported version to 3.9.
```

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -156,3 +156,15 @@ releases:
...
+      - Remove support for ansible-core < 2.14.
+      release_summary: This major release drps support for ansible-core versions lower
```
Comment (hakbailey) [inline changelogs/changelog.yaml:164]:
```suggestion
      release_summary: This major release drops support for ansible-core versions lower
```

**Review by GomathiselviS — APPROVED** [review]

**Review by hakbailey — APPROVED** [review]

**Comment by jillr** [comment]
recheck

**Comment by mariolenz** [comment]
ansible-galaxy-importer failed with:

```
ERROR! Failed to resolve the requested dependencies map. Got the candidate cloud.common:3.0.0 (dependency of vmware.vmware_rest:3.0.0) which didn't satisfy all of the following requirements:
* cloud.common:3.0.0
* cloud.common:>=2.0.4,<3.0.0
Hint: Pre-releases hosted on Galaxy or Automation Hub are not installed by default unless a specific version is requested. To enable pre-releases globally, use --pre.
Starting galaxy collection install process
Process install dependency map
```

Do we have some kind of circular dependency here? Does anyone know where the `vmware.vmware_rest:3.0.0` comes from?

**Comment by GomathiselviS** [comment]
> ansible-galaxy-importer failed with:
> 
> ```
> ERROR! Failed to resolve the requested dependencies map. Got the candidate cloud.common:3.0.0 (dependency of vmware.vmware_rest:3.0.0) which didn't satisfy all of the following requirements:
> * cloud.common:3.0.0
> * cloud.common:>=2.0.4,<3.0.0
> Hint: Pre-releases hosted on Galaxy or Automation Hub are not installed by default unless a specific version is requested. To enable pre-releases globally, use --pre.
> Starting galaxy collection install process
> Process install dependency map
> ```
> 
> Do we have some kind of [circular dependency](https://github.com/ansible-collections/vmware.vmware_rest/pull/446#issuecomment-1814914150) here? Does anyone know where the `vmware.vmware_rest:3.0.0` comes from?

It comes from [here](https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L392). I will work on removing this job since it has already been invoked in the GitHub Actions workflow.

**Comment by GomathiselviS** [comment]
recheck

**Comment by mariolenz** [comment]
Excuse me, but now that this is merged (and tagged 3.0.0) I tried to find it on Galaxy but couldn't.

Isn't this collection published there automatically when you tag it?

**Comment by alinabuzachis** [comment]
> Excuse me, but now that this is merged (and tagged 3.0.0) I tried to find it on Galaxy but couldn't.
> 
> Isn't this collection published there automatically when you tag it?

The job that publishes the collection to Ansible Galaxy failed, however, @jillr will fix this by manually pushing the release.

**Comment by mariolenz** [comment]
@alinabuzachis If the collection _should_ be published automatically, the problem might be related to [Bot used for publishing has lost namespace access](https://forum.ansible.com/t/bot-used-for-publishing-has-lost-namespace-access/1413).

But @jillr should know about this, I think I've seen her somewhere in the thread.

**Comment by jillr** [comment]
They have manually published the release, and fixed the namespace permissions for the bot account on galaxy (we can do that ourselves now, it's super awesome!).

**Comment by mariolenz** [comment]
> **They**

Sorry, I didn't want to be disrespectful. I'll try to remember.

> have manually published the release, and fixed the namespace permissions for the bot account on galaxy

Great, thanks!

**Pull request merged**